packages/renderer/src/lib/dialogs/CommandPalette.spec.ts

```diff
@@ -43,7 +41,7 @@ beforeAll(() => {
   // mock missing scrollIntoView method
   window.HTMLElement.prototype.scrollIntoView = vi.fn();
 
-  (window as any).executeCommand = executeCommandMock;
+  Object.defineProperty(window, 'executeCommand', { value: executeCommandMock });
```

Comment:
Another method (to be aligned with https://github.com/podman-desktop/podman-desktop/blob/main/CODE-GUIDELINES.md#use-vimocked-not-a-generic-myfunctionmock) would be to not declare anymore this mock here (it is done automatically), and use `expect(window.executeCommand).to...` in the tests

Reply:
yes this line should be removed